packages/alfa-rules/src/common/predicate/is-rendered.ts

```diff
@@ -4,8 +4,7 @@ import { Element, Comment, Node } from "@siteimprove/alfa-dom";
 import { Predicate } from "@siteimprove/alfa-predicate";
 import { Context } from "@siteimprove/alfa-selector";
 import { Style } from "@siteimprove/alfa-style";
-
-const { isElement, hasName } = Element;
+import { isFallback } from "../predicate";
```

Comment:
```suggestion

import { isFallback } from "./is-fallback.ts";
```
Here, we should get it from the individual file, to avoid the risk of circular import (`predicate.ts` does import `is-rendered.ts`…